changelogs/current.yaml

```diff
@@ -237,6 +237,11 @@ bug_fixes:
 - area: http2
   change: |
     Update nghttp2 to resolve CVE-2024-30255 https://github.com/envoyproxy/envoy/security/advisories/GHSA-j654-3ccm-vfmm.
+- area: outlier detection
+  change: |
+    Fixed
+    :ref:`successful_active_health_check_uneject_host<envoy_v3_api_field_config.cluster.v3.OutlierDetection.successful_active_health_check_uneject_host>`.
```

Comment:
```suggestion
    :ref:`successful_active_health_check_uneject_host <envoy_v3_api_field_config.cluster.v3.OutlierDetection.successful_active_health_check_uneject_host>`.
```

Reply:
I don't get the intention of this space.
I have changed the line breaks, please check if ok in this way.